src/main/resources/default/templates/biz/login.html.pasta

```diff
@@ -2,12 +2,146 @@
+                                @i18n('Model.login.login')
+                            </button>
+                            <div class="d-flex flex-row justify-content-between">
+                                <a href="/saml" class="link text-small">@i18n('SAMLController.loginViaSAML')</a>
+                                <a href="#" onclick="showForgetPasswordModal()" class="link text-small">
+                                    @i18n('Model.login.forgotPassword')
+                                </a>
+                            </div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <div class="col-12 col-md-7">
+            <i:extensions target="tycho-login" point="sidebar"/>
         </div>
     </div>
-</w:page>
+
+    <i:local name="technologies" value="@part(sirius.biz.tycho.DashboardController.class).getTechStack()"/>
+    <i:if test="technologies.size() > 0">
+        <div class="card mt-5 mb-4">
+            <div class="card-body">
+                <h5 class="card-title text-small font-weight-light mb-4">This service is based on Open Source Software:</h5>
+                <div class="d-flex flex-row overflow-hidden align-items-center">
+                    <i:for type="Tuple" var="technology" items="technologies">
+                        <div class="pl-4 pr-4">
+                            <a href="@technology.getSecond()" target="_blank">
+                                <img src="@technology.getFirst()" style="max-height: 32px"/>
+                            </a>
+                        </div>
+                    </i:for>
+                </div>
+            </div>
+        </div>
+    </i:if>
+
+    <t:modal name="forgotPassword" titleKey="Model.login.forgotPassword">
+        <t:textfield class="required" id="mailField" name="email" value="" labelKey="Model.email"/>
+
+        <i:block name="footer">
+            <a onclick="submitForgotPassword()" class="btn btn-danger">@i18n('Model.login.resetPassword')</a>
+        </i:block>
+    </t:modal>
+    <script type="text/javascript">
+        sirius.ready(function () {
+            const _userField = document.getElementById('user');
+            const _passwordField = document.getElementById('password');
+
+            if (!sirius.isEmpty(_userField.value)) {
+                _passwordField.focus();
+            } else {
+                _userField.focus();
+            }
+
+            try {
+                window.localStorage.setItem('samlPostLoginUri', '@originalUrl');
+            } catch (e) {
+                console.log(e);
+            }
+        });
+
+        function showForgetPasswordModal() {
+            const _emailField = document.getElementById('mailField');
+            const _userField = document.getElementById('user');
+
+            _emailField.value = _userField.value;
+
+            // This bootstrap call sill needs jQuery :(...
```

Comment:
```suggestion
            // This bootstrap call still needs jQuery :(...
```

Reply:
is it available to comment out completely for the browser like <!--@ ?